app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -0,0 +1,143 @@
+package org.wikipedia.feed.suggestededits
+
+
+import android.content.Context
+import android.net.Uri
+import android.text.TextUtils
+import android.view.View
+import android.widget.FrameLayout
+import io.reactivex.annotations.NonNull
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.android.synthetic.main.fragment_add_title_descriptions_item.view.*
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.restbase.page.RbPageSummary
+import org.wikipedia.feed.view.DefaultFeedCardView
+import org.wikipedia.feed.view.FeedAdapter
+import org.wikipedia.page.PageTitle
+import org.wikipedia.util.DimenUtil
+import org.wikipedia.util.ResourceUtil
+import org.wikipedia.util.StringUtil
+import org.wikipedia.views.ItemTouchHelperSwipeAdapter
+
+class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEditsCard>(context), ItemTouchHelperSwipeAdapter.SwipeableView {
+    interface Callback {
+        fun onSuggestedEditsCardClick(@NonNull pageTitle: PageTitle, @NonNull sourceDescription: String, @NonNull sourceLangCode: String, @NonNull view: SuggestedEditsCardView)
+    }
+
+    private val disposables = CompositeDisposable()
+    val CARD_BOTTOM_PADDING = 16.0f
+    val ARTICLE_EXTRACT_MAX_LINE_WITHOUT_IMAGE = 6
+    var translation: Boolean = false
+    private var sourceDescription: String = ""
+    private val app = WikipediaApp.getInstance()
+    private var summary: RbPageSummary? = null
+
+    init {
+        View.inflate(getContext(), R.layout.fragment_add_title_descriptions_item, this)
+    }
+
+    override fun setCard(@NonNull card: SuggestedEditsCard) {
+        super.setCard(card)
+        prepareViews()
+        translation = card.isTranslation()
+        summary = card.getsummary()
+        sourceDescription = card.getSourceDescription()
+        setLayoutDirectionByWikiSite(card.wikiSite(), rootView!!)
```

Comment:
For a "Translation description" card, it should following the source language code for the layout direction.

![Screenshot_1553730849](https://user-images.githubusercontent.com/2435576/55120108-d5540e80-50b1-11e9-9fc2-bfb542e966b0.png)